<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpecialSymbol</class>
 <widget class="QWidget" name="SpecialSymbol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>符号</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_medicalSymbol">
        <attribute name="title">
         <string>医学符号</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTableView" name="tW_medicalSymbol"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_commonSymbol">
        <attribute name="title">
         <string>常用符号</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QTableView" name="tW_commonSymbol"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_numberSymbol">
        <attribute name="title">
         <string>数字符号</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QTableView" name="tW_numberSymbol"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_mathSymbol">
        <attribute name="title">
         <string>数学符号</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTableView" name="tW_mathSymbol"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_specialSymbol">
        <attribute name="title">
         <string>特殊符号</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QTableView" name="tW_specialSymbol"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>368</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Ok">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Cancel">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
